js/script.js

```diff
@@ -15,18 +15,50 @@
 function Generator() {
 
     /* Name Arrays: Customize names to change possible output */
-    this.last_names = ['the Chef', 'Digital', 'Wise', 'Knight', 'Wrecka', 'the Genius', 'the Zoo Keeper', 'the Monk', 'the Scientist', 'the Disciple', 'the Darkman', 'Pellegrino', 'the Ill Figure', 'Rocks The World', 'the Baptist',];
-    this.first_names = ['Inspectah', 'Masta', 'Poppa', 'Five Foot', 'Ghostface', 'Old Dirty'];
+    this.last_names = ['The Dill', 'the Chef', 'Digital', 'Wise', 'Knight', 'Wrecka', 'the Genius', 'the Zoo Keeper', 'the Monk', 'the Scientist', 'the Disciple', 'the Darkman', 'Pellegrino', 'the Ill Figure', 'Rocks The World', 'the Baptist'];
+    this.first_names = ['Fella', 'Inspectah', 'Masta', 'Poppa', 'Five Foot', 'Ghostface', 'Old Dirty'];
 
+    this.generate = function() {
```

Comment:
Admittedly, I wasn't entirely certain about the benefits of defining the method on the prototype instead of the constructor. I would normally just use the constructor for defining all attributes and methods on my objects. So I looked it up and I found this discussion on [StackOverflow](http://stackoverflow.com/questions/4508313/advantages-of-using-prototype-vs-defining-methods-straight-in-the-constructor).

So there are two benefits: 
1. Efficiency => The JS runtime keeps a single copy of method for all Generator objects
2. Stickiness => If the method gets redefined later during execution after the objects are created, the new method definition applies to all existing Generator objects.

Interesting.